### ISSUE TITLE
rc1 findings

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#2882]
### Product versie / Product version

2.1.0.rc1

### Omschrijf het probleem / Describe the bug

- [x] Clicking step 1 highlights step 2 (might have been a fluke- cannot reproduce)
![image](https://user-images.githubusercontent.com/112621537/224955682-5774d847-7d91-4e56-a0a3-76f97870fa77.png)

- [x] Inaccurate logic error/warnings
![image](https://user-images.githubusercontent.com/112621537/224956087-e6ce5bc8-8de1-4c82-8650-20205e28fc43.png)
These are fixed by: anders > nders > anders. Save and the error is gone. 

- [x] Logic Warnings/errors do not display when loading a form, only after the form is saved the errors/warnings appear. Not clear if changes are saved (need to completely reload form, save again to make warnings reappear. Especially annoying when importing a form

- [ ] #3240

Can reproduce: https://our-test-env.nl/zxcczcasdasdzxczxczxccasasdas-2d6a38/startpagina

- [x] validation values missing after import:
![image](https://user-images.githubusercontent.com/112621537/224971736-1f7d8b6b-30d4-4df6-9756-535d21dbfaef.png)

- [x] After pausing the session is not "destroyed" and allows users to click on steps which cause an error:
![image](https://user-images.githubusercontent.com/112621537/224974974-e4fe0695-1931-4875-8d3e-4f8c7ae69d96.png)

**Comment by LaurensBurger** [comment]
@joeribekker only one issue remains, although i believe it's pretty minor and the form builder sees a warning in the logic tab that broke the form.

**Comment by joeribekker** [comment]
Ok, lets create a separate issue for it and close this issue. Thanks @LaurensBurger
